Alerting docs: adds top-level landing page tiles (#83825)
grafana/grafana: docs/sources/alerting/_index.md

menuTitle: Alerting
title: Grafana Alerting
weight: 114
hero:
  title: Grafana Alerting
  level: 1
  image: /media/docs/grafana-cloud/alerting-and-irm/grafana-icon-alerting.svg

>   width: 110
>   height: 110

Reviewer: Results in better spacing before the horizontal rule.
```suggestion
  width: 100
  height: 100
```

  description: Grafana Alerting allows you to learn about problems in your systems moments after they occur.
cards:
  title_class: pt-0 lh-1
  items:
    - title: Introduction

>       href: /docs/grafana-cloud/alerting-and-irm/alerting/

Reviewer: The previous link linked to itself in Cloud and to the Cloud landing page from OSS. I think this is meant to go to the page with the first heading "Introduction". The relative path below works in both Grafana OSS and Cloud.
```suggestion
      href: ./fundamentals/
```

Author: hmm so i need all the links to work from both oss and cloud. should they all be changed?

      description: Learn more about the fundamentals and available features that help you create, manage, and respond to alerts; and improve your team’s ability to resolve issues quickly.
      height: 24
    - title: Set up

>       href: /docs/grafana/next/alerting/set-up/

Reviewer: To work in both OSS and Cloud docs
```suggestion
      href: ./set-up/
```

      description: Set up your implementation of Grafana Alerting.
      height: 24
    - title: Configure alert rules

>       href: /docs/grafana/next/alerting/alerting-rules/

Reviewer: To work in both OSS and Cloud docs
```suggestion
      href: ./alerting-rules/
```

Author: ok..there's my answer haha

      description: Create, manage, view, and adjust alert rules to alert on your metrics data or log entries from multiple data sources — no matter where your data is stored.
      height: 24
    - title: Configure notifications

>       href: /docs/grafana/next/alerting/configure-notifications/

Reviewer: To work in both OSS and Cloud docs
```suggestion
      href: ./configure-notifications/
```

      description: Choose how, when, and where to send your alert notifications.
      height: 24
    - title: Detect and respond

>       href: /docs/grafana/next/alerting/manage-notifications/

Reviewer: To work in both OSS and Cloud docs
```suggestion
      href: ./manage-notifications/
```

      description: Monitor, respond to, and triage issues within your services.
      height: 24
    - title: Monitor

>       href: /docs/grafana/next/alerting/monitor/

Reviewer: To work in both OSS and Cloud docs
```suggestion
      href: ./monitor/
```

> hero:
>   title: Grafana Alerting
>   level: 1
>   image: /media/docs/grafana-cloud/alerting-and-irm/grafana-icon-alerting.svg
>   width: 110
>   height: 110
>   description: Grafana Alerting allows you to learn about problems in your systems moments after they occur.
> cards:
>   title_class: pt-0 lh-1
>   items:
>     - title: Introduction
>       href: /docs/grafana-cloud/alerting-and-irm/alerting/
>       description: Learn more about the fundamentals and available features that help you create, manage, and respond to alerts; and improve your team’s ability to resolve issues quickly.
>       height: 24
>     - title: Set up
>       href: /docs/grafana/next/alerting/set-up/
>       description: Set up your implementation of Grafana Alerting.
>       height: 24
>     - title: Configure alert rules
>       href: /docs/grafana/next/alerting/alerting-rules/
>       description: Create, manage, view, and adjust alert rules to alert on your metrics data or log entries from multiple data sources — no matter where your data is stored.
>       height: 24
>     - title: Configure notifications
>       href: /docs/grafana/next/alerting/configure-notifications/
>       description: Choose how, when, and where to send your alert notifications.
>       height: 24
>     - title: Detect and respond
>       href: /docs/grafana/next/alerting/manage-notifications/
>       description: Monitor, respond to, and triage issues within your services.
>       height: 24
>     - title: Monitor
>       href: /docs/grafana/next/alerting/monitor/
>       description: Monitor your alerting metrics to ensure you identify potential issues before they become critical.
>       height: 24

Reviewer: ```suggestion
hero:
  title: Grafana Alerting
  level: 1
  image: /media/docs/grafana-cloud/alerting-and-irm/grafana-icon-alerting.svg
  width: 100
  height: 100
  description: Grafana Alerting allows you to learn about problems in your systems moments after they occur.
cards:
  title_class: pt-0 lh-1
  items:
    - title: Introduction
      href: ./fundamentals/
      description: Learn more about the fundamentals and available features that help you create, manage, and respond to alerts; and improve your team’s ability to resolve issues quickly.
      height: 24
    - title: Set up
      href: ./set-up/
      description: Set up your implementation of Grafana Alerting.
      height: 24
    - title: Configure alert rules
      href: ./alerting-rules/
      description: Create, manage, view, and adjust alert rules to alert on your metrics data or log entries from multiple data sources — no matter where your data is stored.
      height: 24
    - title: Configure notifications
      href: ./configure-notifications/
      description: Choose how, when, and where to send your alert notifications.
      height: 24
    - title: Detect and respond
      href: ./manage-notifications/
      description: Monitor, respond to, and triage issues within your services.
      height: 24
    - title: Monitor
      href: ./monitor/
      description: Monitor your alerting metrics to ensure you identify potential issues before they become critical.
      height: 24

```
